backend/crypt/cipher.go

```diff
@@ -200,6 +201,15 @@ func newCipher(mode NameEncryptionMode, password, salt string, dirNameEncrypt bo
 	return c, nil
 }
 
+// setEncryptedSuffix set suffix, or an empty string
+func (c *Cipher) setEncryptedSuffix(suffix string) {
+	if !strings.Contains(suffix, ".") {
+		c.encryptedSuffix = ""
```

Comment:
I don't like setting it blank if there isn't a `.` - I think we should probably be adding a `.` and writing an `fs.Errorf` level warning for the user.

Maybe we should have a suffix of `none` to mean don't add a suffix rather than leaving it blank?

Reply:
have updated the "blank" detection and added the error as suggested